### ISSUE TITLE
Update to 2.0.4.35 not offered

### ISSUE THREAD

**Issue by KommX** [duplicati/duplicati#4012]
- [x] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.4.34_canary_2019-11-05
- **Operating system**: Windows 10
- **Backend**: doesn't matter here

## Description

I've seen that there was a new release to the canary channel (2.0.4.35) but I'm not able to update to it like it did with previous versions.

## Steps to reproduce

1. Open Duplicati 2.0.4.34
2. Go to "About" in the main menu
3. Click "Check for updates now"

- **Actual result**:
Nothing happens, just as if there was new version
- **Expected result**:
I should get the offer to update to 2.0.4.35

**Comment by ts678** [comment]
Looks like an update server problem, telling Duplicati 2.0.4.34 is latest:

```
C:\ProgramData\Duplicati\duplicati-2.0.4.33_canary_2019-11-01>Duplicati.Library.AutoUpdater.exe list
 * 2.0.4.33_canary_2019-11-01 (2.0.4.33)

C:\ProgramData\Duplicati\duplicati-2.0.4.33_canary_2019-11-01>Duplicati.Library.AutoUpdater.exe check
New version is available: 2.0.4.34_canary_2019-11-05

C:\ProgramData\Duplicati\duplicati-2.0.4.33_canary_2019-11-01>cd ..\*2.0.4.34*

C:\ProgramData\Duplicati\duplicati-2.0.4.34_canary_2019-11-05>Duplicati.Library.AutoUpdater.exe check
No updates found

C:\ProgramData\Duplicati\duplicati-2.0.4.34_canary_2019-11-05>cd ..\*2.0.4.35*

C:\ProgramData\Duplicati\duplicati-2.0.4.35_canary_2019-11-29>Duplicati.Library.AutoUpdater.exe check
No updates found

C:\ProgramData\Duplicati\duplicati-2.0.4.35_canary_2019-11-29>cd ..\*2.0.4.36*

C:\ProgramData\Duplicati\duplicati-2.0.4.36_canary_2019-12-08>Duplicati.Library.AutoUpdater.exe check
No updates found

C:\ProgramData\Duplicati\duplicati-2.0.4.36_canary_2019-12-08>Duplicati.Library.AutoUpdater.exe list
 * 2.0.4.36_canary_2019-12-08 (2.0.4.36)

C:\ProgramData\Duplicati\duplicati-2.0.4.36_canary_2019-12-08>Duplicati.CommandLine.exe system-info
Duplicati: Duplicati.CommandLine, Version=2.0.4.36, Culture=neutral, PublicKeyToken=null (Duplicati.Library.Main, Version=2.0.4.36, Culture=neutral, PublicKeyToken=null)
Autoupdate urls: https://updates.duplicati.com/canary/latest.manifest;https://alt.updates.duplicati.com/canary/latest.manifest
Update folder: C:\ProgramData\Duplicati\duplicati-2.0.4.36_canary_2019-12-08\updates
Base install folder: C:\ProgramData\Duplicati\duplicati-2.0.4.36_canary_2019-12-08
Version name: "2.0.4.36_canary_2019-12-08" (2.0.4.36)
Current Version folder C:\ProgramData\Duplicati\duplicati-2.0.4.36_canary_2019-12-08
OS: Microsoft Windows NT 10.0.18363.0
```

Download `https://updates.duplicati.com/canary/latest.manifest` as shown above, and it has this:

![image](https://user-images.githubusercontent.com/40771998/70397504-d0857f80-19e0-11ea-85d7-cf4823b31ad2.png)

which looks wrong. At this point, it should be talking about 2.0.4.36, which was released earlier today.

**Comment by warwickmm** [comment]
This should be fixed now.
